### DISCUSSION TITLE
note内のコードブロックが突き抜ける

### DISCUSSION THREAD

**Discussion by oriom-dev** [increments/qiita-discussions#252] [要望・アイデア・不具合]
note内にコードブロックを記述した際、右側が突き抜けてしまう。
<img width="1402" alt="image" src="https://user-images.githubusercontent.com/47474097/179542070-a3b0742f-472f-44bf-ac61-475d8f45e70e.png">

**Comment by tanaka-asami415** [comment]
@Lich496 
ありがとうございます🙏
ノート記法でコードブロックをまだサポートしていないのですが、今後はサポートしたいと考えています🙇

**Reply by tanaka-asami415** [reply]
@Lich496 
大変お待たせしました🙇こちら修正が完了いたしました！
